streams/src/test/java/org/apache/kafka/streams/kstream/internals/InternalStreamsBuilderTest.java

```diff
@@ -156,6 +156,30 @@ public void shouldStillMaterializeSourceKTableIfMaterializedIsntQueryable() thro
         assertEquals("topic2", topology.storeToChangelogTopic().get(storeName));
         assertNull(table1.queryableStoreName());
     }
+    
+    @Test
+    public void shouldBuildGlobalTableWithNonQueryableStoreName() throws Exception {
+        final GlobalKTable<String, String> table1 = builder.globalTable("topic2",
```

Comment:
nit: move `"topic2"` to next line and fix indention (should only indent by 4 spaced) -- this will reduce the line length and make code better readable.